annex/chapter1/adv_examples/include/position.h

```diff
@@ -0,0 +1,15 @@
+#include <numeric>
```

Comment:
Add header include guard on the first line of the file similar to other header files.

Reply:
Done